ci/prow/make_config.go

```diff
@@ -845,6 +873,20 @@ func generateCleanupPeriodicJob() {
 	executeJobTemplate("periodic cleanup", cleanupPeriodicJob, "presubmits", "", data.PeriodicJobName, false, data)
 }
 
+// generateFlakyTestReporterPeriodicJob generates the flaky tests reporter job config.
+func generateFlakyTestReporterPeriodicJob() {
+	var data periodicJobTemplateData
+	data.Base = newbaseProwJobTemplateData("knative/test-infra")
+	data.PeriodicJobName = "ci-knative-flakytestreporter"
```

Comment:
Can we just use "flakes" instead of the long "flakytestreporter"?

Reply:
renamed to "flakyTool"